src/lib/select/select.spec.ts

```diff
@@ -2690,6 +2690,41 @@ describe('MatSelect', () => {
 
   });
 
+  describe('with option centering disabled', () => {
+    beforeEach(async(() => configureMatSelectTestingModule([
+      SelectWithoutOptionCentering,
+    ])));
+
+    let fixture: ComponentFixture<SelectWithoutOptionCentering>;
+    let trigger: HTMLElement;
+    let select: HTMLElement;
+    let formField: HTMLElement;
+
+    beforeEach(fakeAsync(() => {
+      fixture = TestBed.createComponent(SelectWithoutOptionCentering);
+      fixture.detectChanges();
+      trigger = fixture.debugElement.query(By.css('.mat-select-trigger')).nativeElement;
+      select = fixture.debugElement.query(By.css('mat-select')).nativeElement;
+      formField = fixture.debugElement.query(By.css('mat-form-field')).nativeElement;
+    }));
+
+    it('should align the first option with trigger text if option centering is disabled',
```

Comment:
"Pizza" is the second option in this case. Also it may be clearer if it was something like `it('should not align the active option with the trigger if centering is disabled')`.